el-debate/src/test/java/pl/elpassion/elspace/debate/chat/TimeFormatterTest.kt

```diff
@@ -0,0 +1,26 @@
+package pl.elpassion.elspace.debate.chat
+
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import java.util.*
+
+class TimeFormatterTest {
+
+    @Test
+    fun shouldReturnCorrectTime() {
+        val time = 50000000L.getTime(TimeZone.getTimeZone("GMT"))
+        assertTrue(time == "13:53")
```

Comment:
Change to assertEqual. This method returns zero information about inputs and outputs on fail.